annotation/core/models.py

```diff
@@ -71,15 +71,23 @@ def boundary(self):
     def __str__(self):
         return self.body
 
-
+class FeedbackOption(models.Model):
+    """The types of reasons people give for thinking sentence is machine generated text"""
+    shortname = models.CharField(max_length=100, primary_key=True)
+    description = models.CharField(max_length=250)
+    category = models.CharField(max_length=100)
```

Comment:
Maybe make category and shortname only 20 character long if we do expect them to always be shorter than that.

Reply:
okay, i wasn't sure if the hash function always returned length <= 20 so i made shortname 40 for now..